src/js/modules/setup/services/login/login.service.js

```diff
@@ -110,9 +116,9 @@
         return self._launchService.storeAccountInfo(accountInfo)
             .then(function() {
                 return self._setupService.setUserInfo({
-                    username: data.responseData.username,
-                    displayName: data.responseData.username,
-                    email: data.responseData.email
+                    username:       data.responseData.username,
+                    displayName:    data.responseData.username,
+                    email:          data.responseData.email
```

Comment:
we're trying to get rid of this style, don't add more of it ;)

Reply:
done